apps/playground/src/app/components/indicators/wait/wait.component.html

```diff
@@ -1,39 +1,36 @@
 <div class="sky-wait-demo">
   <button
-    class="sky-btn sky-btn-secondary sky-test-wait"
+    class="sky-btn sky-btn-primary sky-test-wait sky-margin-inline-sm"
     type="button"
-    [ngClass]="{
-      'sky-btn-secondary': !isWaiting,
-      'sky-btn-primary': isWaiting
-    }"
     (click)="isWaiting = !isWaiting"
   >
     Toggle wait
   </button>
 
   <button
-    class="sky-btn sky-btn-secondary sky-test-full-page"
+    class="sky-btn sky-btn-primary sky-test-full-page sky-margin-inline-sm"
     type="button"
-    [ngClass]="{
-      'sky-btn-secondary': !isFullPageWaiting,
-      'sky-btn-primary': isFullPageWaiting
-    }"
-    (click)="isFullPageWaiting = !isFullPageWaiting"
+    (click)="toggleFullPageWait()"
   >
-    Toggle full page wait
+    Toggle full page wait (5 seconds)
   </button>
 
   <button
-    class="sky-btn sky-btn-secondary sky-test-non-blocking"
+    class="sky-btn sky-btn-primary sky-test-full-page sky-margin-inline-sm"
```

Comment:
I added a button here to toggle a wait via the service - the other changes are just cosmetic.